util/dvsim/CfgFactory.py

```diff
@@ -82,7 +83,11 @@ def make_cfg(path, args, proj_root):
     of the project.
 
     '''
-    initial_values = {'proj_root': proj_root}
+    initial_values = {
+        'proj_root': proj_root,
+        'self_dir': os.path.dirname(path),
+
```

Comment:
super nit: remove empty newline?

Reply:
Indeed, I just saw it a minute ago as well and will push an update. Thanks!